commune_app/templates/commune_app/index.html

```diff
@@ -6,93 +6,110 @@
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.min.js" integrity="sha384-cuYeSxntonz0PPNlHhBs68uyIAVpIIOZZ5JqeqvYYIcEL727kskC66kF92t6Xl2V" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-kenU1KFdBIe4zVF0s0G1M5b4hcpxyD9F7jL+jjXkk+Q2h455rYXK/7HAuoJl+0I4" crossorigin="anonymous"></script>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"/>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"/>
 </head>
 <body>
-    <!-- home section start -->
-    <header class="home">
-        <h1>Commune</h1>
-    </header>
+    <!--NAVIGATION BAR-->
+    <nav class="navbar sticky-top navbar-light navbar-expand-lg">
+        <div class="container max-width">
+        <a href="#" class="navbar-brand">
+        <img class="logo" src="{% static 'images/Communet.png' %}" alt="Logo" title="Logo"></a>
+        <button class="navbar-toggler bg-light" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-label="navbar">
+        <span class="navbar-toggler-icon bg-light">
+        </span></button>
+        <div class="collapse navbar-collapse" id="navbarResponsive">
+            <ul class="navbar-nav ml-auto">
+            <li class="nav-item"><a href="{% url 'Main_Page' %}" class="nav-link active text-white">Home</a></li>
+            <li class="nav-item"><a href="#story" class="nav-link active text-white">About Us</a></li>
+            <li class="nav-item"><a href="{% url 'chores' %}" class="nav-link active text-white">Chores</a></li>
+            <li class="nav-item"><a href="#team" class="nav-link active text-white">Team</a></li></ul>
+        </div>
+        </div>
+    </nav>
     
-    <main class="container">
-        <section class="about-us">
-            <div class="ab-container">
-                <div class="row">
-                    <div class="col-sm-5">
-                        <h1 class="text-center about-us-h">About us</h1>
-                        <p class="about-us-p">The Communet platform allows people with shared interests and values to create a financially cooperative community, with pre-established norms and procedures.
+    <!--THE HERO IMAGE PIC -->
+    <section class="hero-image">
+        <div class="max-width">
+          <div class="hero-text">
+            <h1>Communet</h1>
+            <p>Bring People Together</p>
+            <!--<button></button>-->
+          </div>
+        </div>
+    </section> 
 
-    Chores, shopping, and bills are all pre-configured and agreed upon before co-habitation begins, taking the mess and complexity out of starting a shared life.
+    <!---about us page-->
+    <section class="aboutus" id="story">
```

Comment:
all pages must have one `<main>` landmark

Reply:
i have for each one section to make it easier for people to read and i added also main landmark